### ISSUE TITLE
AttributeError: module 'wx' has no attribute 'LANGUAGE_CZECH_CZECHIA'

### ISSUE THREAD

**Issue by MCUmbrella** [jeanslack/Videomass#354]
## Thanks for taking the time to submit a bug report! Please include the following information:   

**What operating system are you using?**   
Kubuntu 21.10

**What version of Videomass are you using and how did you install or run it?**   
```
Commandline: apt remove python3-videomass
Remove: python3-videomass:amd64 (5.0.11-1)

Commandline: apt install ./videomass_5.0.18-1_all.deb
Install: videomass:amd64 (5.0.18-1)
```

**Describe the bug**   
```
$ videomass 
Traceback (most recent call last):
  File "/usr/bin/videomass", line 5, in <module>
    from videomass.gui_app import main
  File "/usr/lib/python3/dist-packages/videomass/gui_app.py", line 38, in <module>
    from videomass.vdms_sys import app_const as appC
  File "/usr/lib/python3/dist-packages/videomass/vdms_sys/app_const.py", line 25, in <module>
    "cs_CZ": (wx.LANGUAGE_CZECH_CZECHIA, ("Czech (Czechia)")),
AttributeError: module 'wx' has no attribute 'LANGUAGE_CZECH_CZECHIA'
```

**Comment by ThoreauHenry** [comment]
Same issue on 22.04/Jammy.

`./videomass
Traceback (most recent call last):
  File "/usr/bin/./videomass", line 5, in <module>
    from videomass.gui_app import main
  File "/usr/lib/python3/dist-packages/videomass/gui_app.py", line 38, in <module>
    from videomass.vdms_sys import app_const as appC
  File "/usr/lib/python3/dist-packages/videomass/vdms_sys/app_const.py", line 25, in <module>
    "cs_CZ": (wx.LANGUAGE_CZECH_CZECHIA, ("Czech (Czechia)")),
AttributeError: module 'wx' has no attribute 'LANGUAGE_CZECH_CZECHIA'
Error in sys.excepthook:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/apport_python_hook.py", line 153, in apport_excepthook
    with os.fdopen(os.open(pr_filename,
FileNotFoundError: [Errno 2] No such file or directory: '/var/crash/_usr_bin_videomass.1000.crash'

Original exception was:
Traceback (most recent call last):
  File "/usr/bin/./videomass", line 5, in <module>
    from videomass.gui_app import main
  File "/usr/lib/python3/dist-packages/videomass/gui_app.py", line 38, in <module>
    from videomass.vdms_sys import app_const as appC
  File "/usr/lib/python3/dist-packages/videomass/vdms_sys/app_const.py", line 25, in <module>
    "cs_CZ": (wx.LANGUAGE_CZECH_CZECHIA, ("Czech (Czechia)")),
AttributeError: module 'wx' has no attribute 'LANGUAGE_CZECH_CZECHIA'
`

**Comment by jeanslack** [comment]
@MCUmbrella @ThoreauHenry  
Thank you for reporting.

Please, can you try changing `wx.LANGUAGE_CZECH_CZECHIA` constant to `wx.LANGUAGE_CZECH` in the file `"/usr/lib/python3/dist-packages/videomass/vdms_sys/app_const.py"` and see if that traceback disappears?

I also tried it on a Linux box (Slackware) with all the languages ​​installed and it works by setting the language to Czech from Videomass Preferences.

**Comment by ThoreauHenry** [comment]
$ videomass
Traceback (most recent call last):
  File "/usr/bin/videomass", line 5, in <module>
    from videomass.gui_app import main
  File "/usr/lib/python3/dist-packages/videomass/gui_app.py", line 38, in <module>
    from videomass.vdms_sys import app_const as appC
  File "/usr/lib/python3/dist-packages/videomass/vdms_sys/app_const.py", line 30, in <module>
    "hu_HU": (wx.LANGUAGE_HUNGARIAN_HUNGARY, ("Hungarian (Hungary)")),
AttributeError: module 'wx' has no attribute 'LANGUAGE_HUNGARIAN_HUNGARY'
Error in sys.excepthook:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/apport_python_hook.py", line 153, in apport_excepthook
    with os.fdopen(os.open(pr_filename,
FileNotFoundError: [Errno 2] No such file or directory: '/var/crash/_usr_bin_videomass.1000.crash'

Original exception was:
Traceback (most recent call last):
  File "/usr/bin/videomass", line 5, in <module>
    from videomass.gui_app import main
  File "/usr/lib/python3/dist-packages/videomass/gui_app.py", line 38, in <module>
    from videomass.vdms_sys import app_const as appC
  File "/usr/lib/python3/dist-packages/videomass/vdms_sys/app_const.py", line 30, in <module>
    "hu_HU": (wx.LANGUAGE_HUNGARIAN_HUNGARY, ("Hungarian (Hungary)")),
AttributeError: module 'wx' has no attribute 'LANGUAGE_HUNGARIAN_HUNGARY'

It seems to not like Country names.

**Comment by jeanslack** [comment]
After some tests I report this conclusion: Your problem is due to a too old version of wxPython (specifically `python3-wxgtk4.0` package v4.0.7 available on Ubuntu 21.10 and 22.04/Jammy) which does not support all the identifiers listed on this [page](https://docs.wxpython.org/wx.Language.enumeration.html).

As a workaround, I propose to open the file "/usr/lib/python3/dist-packages/videomass/vdms_sys/app_const.py" and replace the constants `wx.LANGUAGE_CZECH_CZECHIA`,` wx.LANGUAGE_HUNGARIAN_HUNGARY`, `wx.LANGUAGE_SPANISH_CUBA` with the following `wx.LANGUAGE_CZECH`, `wx.LANGUAGE_HUNGARIAN`, `wx.LANGUAGE_SPANISH`.

As a definitive solution I recommend you to install a more up-to-date version of wxPython or to upgrade your system to a more up-to-date version of Ubuntu, for example K/Ubuntu 24.04/24.10 which already provides one of the latest versions of wxPython.
